src/App.tsx

```diff
@@ -1,7 +1,12 @@
+import { BrowserRouter } from "react-router-dom";
+import AppRoutes from "./app/app-routes";
+
 function App() {
   return (
     <div className="App">
-
+      <BrowserRouter>
```

Comment:
The intent was to have a file dedicated to hosting the routes themselves, separated from the actual router - this would also help minimizing the code in App

I moved the routes within `App.tsx` 👍